pages/index.tsx

```diff
@@ -31,7 +36,11 @@ const Home: NextPage = () => {
 
       <HeroSection onHandleGetStarted={handleGetStarted} />
       <HomeCarousel />
-      <Footer />
+      <MidSection/>
```

Comment:
Everything looks great! Just need to swap these two, the mid section should come before the carousel. Also need to fix the mobile version, should be stacked in a column and full-width on small screens:

![small-screen](https://user-images.githubusercontent.com/20467669/177612487-fb97a500-6713-4115-b807-1bce8acfce16.png)